script/validate-competencies-json.js

```diff
@@ -33,9 +35,18 @@ validateJsonFiles([
 async function validateJsonFiles(configurations) {
 	let errors = [];
 	for (const configuration of configurations) {
-		errors = errors.concat(
-			await validateJsonFileUsingSchema(configuration.path, configuration.schema)
-		);
+		try {
+			errors = errors.concat(
+				await validateJsonFileUsingSchema(configuration.path, require(configuration.schema))
+			);
+		} catch (error) {
+			errors.push({
+				path: '',
+				filePath: configuration.schema,
+				expected: 'No errors in loading',
+				received: error.message
+			});
```

Comment:
This error object matches the errors returned by our JSON Schema library. It's not ideal that we're just constructing an object like this but as it's in one place it makes little sense to abstract it away.